ajax/libs/angular-material-calendar/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "angular-material-calendar",
+  "version": "0.2.14",
+  "description": "A calendar directive for AngularJS and Angular Material Design",
+  "keywords": [
+    "angular",
+    "material",
+    "design",
+    "calendar",
+    "ngCalendar",
+    "mdCalendar",
+    "calendarMd",
+    "calendar-md",
+    "md-calendar",
+    "ng-calendar"
+  ],
+  "homepage": "https://angular-material-calendar.bradb.net/",
+  "filename": "angular-material-calendar.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/bradberger/material-calendar.git"
+  },
+  "authors": [
+    "Brad Berger <brad@bradb.net>",
+    "Gavin Barron <gavin.barron@teamintergen.com>"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/bradberger/material-calendar.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "angular-material-calendar*.*(js|css|map)"
```

Comment:
Let's use exclude pattern here `angular-material-calendar!(*.html)`
Because in `dist`, `build` or `release` folder, we trust all the files which under these folders can be added.
Except there are some files that we can't use in web front-end, then we will exclude them.
Many thanks.

Memo: this pattern exclude `angular-material-calendar.html` file.